src/e2e-test/java/io/cdap/plugin/bigquerymultitable/actions/BigQueryMultiTableActions.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.bigquerymultitable.actions;
+
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.plugin.bigquerymultitable.locators.BigQueryMultiTableLocators;
+
+/**
+ * BigQueryMultiTable plugin step actions.
+ */
+public class BigQueryMultiTableActions {
+
+  static {
+    SeleniumHelper.getPropertiesLocators(BigQueryMultiTableLocators.class);
+  }
+
+    public static void enterSqlStatement(String value) {
```

Comment:
Move the locator and the corresponding action to the framework repo, if it's not already there.